### ISSUE TITLE
Update for the new TTS API

### ISSUE THREAD

**Issue by pavlukivan** [zlargon/google-tts#35]
https://github.com/Boudewijn26/gTTS-token/blob/master/docs/november-2020-translate-changes.md

**Comment by ncpierson** [comment]
It looks like this change went 100% today. All requests are failing for me. Solving this will solve #33 though I don't know how difficult it will be.

**Comment by freddiefujiwara** [comment]
@ncpierson 
If you are in a hurry please use the following package instead of this one
https://github.com/freddiefujiwara/google-tts/tree/feature/fix-dec-1

in your package.json

```JavaScript
"dependencies": {
        ...
        "google-tts-api": "https://github.com/freddiefujiwara/google-tts#feature/fix-dec-1",
       ...
    },
```

**Comment by ncpierson** [comment]
I was going to contribute to this repo but I decided to just do a rewrite since so much code would have to change:
https://github.com/ncpierson/google-translate-tts

**Comment by zlargon** [comment]
New Google TTS API has been implemented in version 2.0.0, and there are some breaking changes.

Please see more details in [README](https://github.com/zlargon/google-tts/blob/master/README.md) and [CHANGELOG](https://github.com/zlargon/google-tts/blob/master/CHANGELOG.md)
Thanks.


```js
const fs = require('fs');
const googleTTS = require('google-tts-api');

// get base64 text
googleTTS
  .getAudioBase64('Hello World', { lang: 'en-US', slow: false })
  .then((base64) => {
    console.log({ base64 });

    // save the audio file
    const buffer = Buffer.from(base64, 'base64');
    fs.writeFileSync('hello-world-english.mp3', buffer, { encoding: 'base64' });
  })
  .catch(console.error);
```
